Windows: module not found error when running CLI after WSL setup

## Problem

Users on Windows who run `setup.sh` via WSL and then try to run `node .mex/dist/cli.js check` from a native Windows terminal get a module not found error.

## Steps to reproduce

1. Clone mex on a Windows machine
2. Run `setup.sh` via WSL
3. Run `node .mex/dist/cli.js check` from the project root (outside WSL)

## What happens

```
Error: Cannot find module '...'
```

## Why

The CLI gets built inside WSL but `node_modules` and path resolution break when the build environment and runtime environment don't match. Mixing Windows filesystem access with WSL tooling causes symlink and path separator issues in `node_modules`.

## Workaround

Run everything inside WSL, don't mix environments:

```bash
cd .mex
npm install
npm run build
cd ..
node .mex/dist/cli.js check
```

## What needs to be fixed

- [ ] Add a prominent Windows callout in the README near the install section: "Windows users: run all commands inside WSL or Git Bash. Do not mix environments."
- [ ] Investigate a native Windows setup path (either a `setup.ps1` PowerShell script or making the WSL flow work end to end without manual rebuilds)
- [ ] Test CLI commands natively on Windows with Node without WSL

--- a/README.md
+++ b/README.md
@@ -86,6 +86,14 @@ Install globally later at any time:
 npm install -g mex-agent
 ```
 
+### Windows
+
+The recommended `npx mex-agent setup` flow runs in any terminal (Command Prompt, PowerShell, or WSL) and does not need bash, so most Windows users do not have to think about this section.
+
+If you previously installed via the legacy `setup.sh` script, do not mix environments. Building inside WSL and then running the CLI from a native Windows terminal causes "module not found" errors because `node_modules` and path resolution differ between the two filesystems. Run install, build, and CLI commands inside the same environment: either entirely in WSL / Git Bash, or entirely in native Windows via `npx mex-agent`.
+
+See [issue #10](https://github.com/theDakshJaitly/mex/issues/10) for context.
+
 ## How It Works
 
 ![mex context routing flow](docs/diagrams/context-routing.svg)

--- a/README.md
+++ b/README.md
@@ -90,7 +90,9 @@ npm install -g mex-agent
 
 The recommended `npx mex-agent setup` flow runs in any terminal (Command Prompt, PowerShell, or WSL) and does not need bash, so most Windows users do not have to think about this section.
 
-If you previously installed via the legacy `setup.sh` script, do not mix environments. Building inside WSL and then running the CLI from a native Windows terminal causes "module not found" errors because `node_modules` and path resolution differ between the two filesystems. Run install, build, and CLI commands inside the same environment: either entirely in WSL / Git Bash, or entirely in native Windows via `npx mex-agent`.
+> **Windows users (legacy `setup.sh` flow):** Run all commands inside WSL or Git Bash. Do not mix environments.
+
+If you previously installed via the legacy `setup.sh` script, building inside WSL and then running the CLI from a native Windows terminal causes "module not found" errors because `node_modules` and path resolution differ between the two filesystems. Run install, build, and CLI commands inside the same environment: either entirely in WSL / Git Bash, or entirely in native Windows via `npx mex-agent`.
 
 See [issue #10](https://github.com/theDakshJaitly/mex/issues/10) for context.
 

